README.md

```diff
@@ -5,7 +5,9 @@
 Here is an example:
 
 ```erlang
-Client = aws_client:make_client(<<"access-key-id">>,
+application:ensure_all_started(aws).
+{ok,[erlware_commons,idna,hackney,jsx,aws]}
```

Comment:
That line is probably not something you want in the code sample -- looks like the return value from the shell instead.